hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -119,7 +119,7 @@ public void toCsvFile(ServicesState signedState, Instant when) {
 					expectedFloat));
 		}
 
-		var csvLoc = lastUsedExportDir + when + "_Balances.csv";
+		var csvLoc = lastUsedExportDir + (when + "_Balances.csv").replace(":", "_");
```

Comment:
I think `DateTimeFormatter.ofPattern(expectedPattern).format(when)` will show a clearer expectation.

Reply:
If we did not touch this class for v0.9.0, the `Instant` class (for `when`) was changing its `toString()` method. And this could happen again in the future, if we don't use a pre-chosen pattern.

Reply:
`Instant.toString()` didn't change, I just left off the `.replace(":", "_")` while refactoring for token balances. 😞 

But agreed it's clearer to use an explicit `DateTimeFormatter`, updated to use `ISO_INSTANT.format()`. 👍

Reply:
I was actually thinking of `DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH_mm_ss.SSSSSSSSS'Z'").format(when)` to be very explicit. But it seems that the ISO_INSTANT comes with the nanosecond resolution while other pattern seems very tricky to obtain that.